GameData/RealismOverhaul/Waterfall_Configs/_Processor/10_initialize.cfg

```diff
@@ -14,5 +14,16 @@
         &rotation = 0,0,0
         &scale = 1,1,1
         // Optional key: `glow`.
+
+        __is_first = true
     }
+
+    // Prevent infinite loop if the part has multiple `ROWaterfall` nodes defined for some reason,
+    // and the subsequent nodes contain `ExtraTemplate`s or `MainPlumeVariant`s.
+    @ROWaterfall:HAS[~__is_first],*
+    {
+        |_ = ROWaterfall_CannotProcess
+    }
+
+    @ROWaterfall,0 { !__is_first = delete }  // Cleanup.
```

Comment:
At this point the `,0` is redundant right?

Reply:
Yes -- would you think it's better removed? I just wanted to be explicit.

Reply:
Yeah, probably should remove.  `,0` suggests I should look for the others.
Also fix the :HAS + ,* in line 23 (remove the ,*)

Reply:
Already removed the `:HAS` ,* 😉 

But sure, I'll remove all of them -- don't think there are many, if at all.